packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManagerImpl.kt

```diff
@@ -81,6 +81,7 @@ internal object LottieAnimationViewManagerImpl {
 
     @JvmStatic
     fun play(view: LottieAnimationView, startFrame: Int, endFrame: Int) {
+        val restartAnimation = startFrame != -1 && endFrame != -1
         Handler(Looper.getMainLooper()).post {
             if (startFrame != -1 && endFrame != -1) {
```

Comment:
Done. I also renamed the variable since it is not exactly restarting the animation and didnt seem clear to me coming to the pr after some time. Hopefully you agree with the new name.